pkg/limayaml/defaults.go

```diff
@@ -195,11 +195,14 @@ func FillPortForwardDefaults(rule *PortForward, instDir string) {
 			home, _ := os.UserHomeDir()
 			limaHome, _ := dirnames.LimaDir()
 			data := map[string]string{
-				"Home":     home,
-				"Instance": filepath.Base(instDir),
-				"LimaHome": limaHome,
-				"UID":      user.Uid,
-				"User":     user.Username,
+				"Home":        home,
+				"InstanceDir": instDir,
+				"Name":        filepath.Base(instDir),
+				"UID":         user.Uid,
+				"User":        user.Username,
+
+				"Instance": filepath.Base(instDir), // DEPRECATED, use `{{.Name}}`
+				"LimaHome": limaHome,               // DEPRECATED, (use `InstanceDir` instead of `{{.LimaHome}}/{{.Instance}}`
```

Comment:
Can we print warnings when deprecated variables are used? (Can be another PR)

Reply:
idk, seems like a lot of effort for little gain: how do you detect that the variable was used in a template expansion?

When I wrote "DEPRECATED" I just meant that we discourage people from using it (by not mentioning it in the docs/sample lima.yaml), and having the comments in the source.

But the variables will still work, and be correct, and I don't see any reason why we would ever have to remove them; they are not in the way of implementing some other feature. It is just that there should be no good reason why a user would ever need them, so there is no reason to make them aware that they exist for backwards compatibility reasons only.

So I think a warning is not really necessary.

Reply:
I already merged, but we could change the comment from "DEPRECATED" to "HIDDEN" if that sounds better?

I would just sneak it in via another PR...